packages/lab/src/list-next/ListItemNext.css

```
@@ -0,0 +1,53 @@
+/* Default variables applied to the root element */
+.saltListItemNext {
+  --listItemNext-text-color: var(--salt-text-primary-foreground);
+  --listItemNext-background: none;
+}
+
+.saltListItemNext {
+  color: var(--listItemNext-text-color);
+  background: var(--listItemNext-background);
+  height: var(--salt-size-stackable, auto);
```

```suggestion
  height: calc(var(--salt-size-base) + var(--salt-spacing-100));
```
this spacing addition might supposed to be padding, would check with designer

---

changed and i will double check with Ivan